src/content/app/tools/blast/views/blast-submission-results/components/single-blast-job-result/SingleBlastJobResult.tsx

```diff
@@ -182,38 +182,52 @@ const SingleBlastJobResult = (props: SingleBlastJobResultProps) => {
 
   const alignmentsCount = countAlignments(jobResult.data);
 
-  return (
-    <div className={styles.resultsSummaryRow}>
-      <div className={styles.hitLabel}>
-        <span>{alignmentsCount} </span>
-        <span>{pluralise('hit', alignmentsCount)}</span>
-      </div>
-      <div className={styles.summaryPlot}>
-        {shouldUseGenomicHitsDiagram ? (
-          <BlastGenomicHitsDiagram
-            genomeId={speciesInfo.genome_id}
-            job={jobResult.data}
-            width={diagramWidth}
-          />
-        ) : (
-          <BlastHitsDiagram job={jobResult.data} width={diagramWidth} />
-        )}
+  if (jobResult.status === 'FAILURE') {
+    return (
+      <div className={styles.failedJobSummaryRow}>
+        <div className={styles.failedJobStatus}>Job failed</div>
+        <BlastSpecies
+          species={speciesInfo}
+          isExpanded={false}
+          toggleExpanded={setExpanded}
+          jobResult={jobResult}
+        />
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/212078530-60d78bde-7fff-4e33-a2eb-960ef6d67b7f.png)

Reply:
This is not to worry. Its because your results data:[] is not empty. The code current code handles it and wont be displayed when the data is empty.